src/Builder/CliMenuBuilder.php

```diff
@@ -395,6 +407,20 @@ public function setSelectedMarker(string $marker) : self
         return $this;
     }
 
+    public function setUntoggledMarker(string $marker) : self
```

Comment:
I think this should rather be `setUncheckedMarker` and `setCheckedMarker`. Toggle to me is the act of changing states, so toggling between something rather than being toggled/untoggled. I could be wrong though.

Reply:
Should I rename the class `CheckableItem`?

I had planned on building on top of this and adding radio-style support as well, where items in a splititem group would toggle other items off once checked, just like a regular radio element.

Reply:
Yeah that sounds fine to me. Also the radio split item sounds very nice.

Reply:
The reason for `ToggleableItem` vs `CheckableItem` had been the future radio functionality. Unless you'd be ok with having two separate classes `CheckableItem` and `RadioItem` or similar and having them share a bunch of code via traits?

Reply:
Yeah I think I'd prefer two separate classes. Trait sounds fine.